cmd/cli/app/history/history_list.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package history
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+
+	"github.com/stacklok/minder/cmd/cli/app"
+	"github.com/stacklok/minder/cmd/cli/app/common"
+	"github.com/stacklok/minder/internal/db"
+	"github.com/stacklok/minder/internal/util"
+	"github.com/stacklok/minder/internal/util/cli"
+	"github.com/stacklok/minder/internal/util/cli/table"
+	"github.com/stacklok/minder/internal/util/cli/table/layouts"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+)
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List history",
+	Long:  `The history list subcommand lets you list history within Minder.`,
+	RunE:  cli.GRPCClientWrapRunE(listCommand),
+}
+
+// listCommand is the profile "list" subcommand
+func listCommand(ctx context.Context, cmd *cobra.Command, _ []string, conn *grpc.ClientConn) error {
+	client := minderv1.NewEvalResultsServiceClient(conn)
+	//client := minderv1.NewProfileServiceClient(conn)
+
+	project := viper.GetString("project")
+	profileName := viper.GetString("profileName")
+	entityName := viper.GetString("entityName")
+	entityType := viper.GetString("entityType")
+	evalStatus := viper.GetString("evalStatus")
+	remediationStatus := viper.GetString("remediationStatus")
+	alertStatus := viper.GetString("alertStatus")
```

Comment:
**polish:** these should be `GetStringSlice` as it is possible to specify multiple entries for each filter.

Luckily, `GetStringSlice` allows the user to specify multiple times the flag like `--entity-type repository --entity-type pull_request`.

Also, elsewhere in the CLI code we have flags styled as `--foo-bar` rather than `--fooBar`.